### PR TITLE
Added new portcall service types.

### PR DESCRIPTION
**Pull request by douradofrazer**

### CODE REVIEW

File: datamodel/referencedata.d/portcallservicetypecodes.csv [hunk 1]
```diff
@@ -4,8 +4,16 @@ MOOR,Mooring,"The activity of attaching a vessel to a berth using mooring lines.
...
 TOWG,Towage,"The activity of one vessel (boat/tug) assisting another vessel with manoeuvres within the port area."
 BUNK,Bunkering,"The transfer of liquid or gaseous fuel from land-based or floating facilities into a ship’s permanent tanks"
-WSDP,Waste disposal,"The activity of removing waste from a vessel to shore during port stay"
 LASH,Lashing,"All activities encompassing securing the containers and other arrangements onboard to prevent them from moving around on the vessel at sea"
 SAFE,Safety,"A safety timestamps indicates an acknowledgement of a participant that the vessel is ready for a responsibility handover from carrier to terminal and otherwise"
 FAST,Fast,"FAST is all fast (last line secured)"
 GWAY,Gangway,"GWAY is gangway down and secure (authorized personnel can board the vessel)"
+ANCO,Anchorage ops,""
+SLUG,Sludge,""
+CONV,Convoy,""
```
Comment (nt-as) [inline datamodel/referencedata.d/portcallservicetypecodes.csv:13]: Not included in JIT

File: datamodel/referencedata.d/portcallservicetypecodes.csv [hunk 2]
```diff
@@ -4,8 +4,16 @@ MOOR,Mooring,"The activity of attaching a vessel to a berth using mooring lines.
...
+ICEN,"Ice navigation",""
```
Comment (nt-as) [inline datamodel/referencedata.d/portcallservicetypecodes.csv:14]: Not included in JIT

File: datamodel/referencedata.d/portcallservicetypecodes.csv [hunk 3]
```diff
@@ -4,8 +4,16 @@ MOOR,Mooring,"The activity of attaching a vessel to a berth using mooring lines.
...
+ESCO,Escorting,""
```
Comment (nt-as) [inline datamodel/referencedata.d/portcallservicetypecodes.csv:15]: Not included in JIT

**Review by HenrikHL — APPROVED** [review]

**Comment by HenrikHL** [comment]
We can update the descriptions later...

**Pull request merged**